hawc/apps/animalv2/models.py

```diff
@@ -36,15 +38,45 @@ class Experiment(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     last_updated = models.DateTimeField(auto_now=True)
 
+    BREADCRUMB_PARENT = "study"
+
     def __str__(self):
         return self.name
 
+    def get_absolute_url(self):
+        return reverse("animalv2:experiment_detail", args=(self.pk,))
+
+    def get_update_url(self):
+        return reverse("animalv2:experiment_update", args=(self.pk,))
+
+    def get_delete_url(self):
+        return reverse("animalv2:experiment_delete", args=(self.pk,))
+
     def get_assessment(self):
         return self.study.get_assessment()
 
     def get_study(self):
         return self.study
 
+    def get_has_multiple_generations_display(self) -> str:
+        return "Yes" if self.has_multiple_generations else "No"
+
+    @property
+    def v2_timepoints(self):
+        timepoints = []
+        for endpoint in self.v2_endpoints.all():
+            timepoints.extend(endpoint.v2_timepoints.all())
+        return timepoints
+        """
```

Comment:
remove comment and make a method not a property; change to something like this `Observation.object.filter(endpoint__experiment=self).order_by('...')`

Reply:
honestly can't recall why I did it this way in the first place! Might have been a remnant of some avenue I backtracked out of. Who knows, anyway, changes made.